docker/debian.docker

```diff
@@ -0,0 +1,25 @@
+FROM fleshgrinder/debian:stretch-go1-build
+COPY . /usr/local/src/github.com/trivago/gollum
+WORKDIR /usr/local/src/github.com/trivago/gollum
+RUN make -sj$(nproc) current
+LABEL com.trivago.image="gollum"
+
+# ------------------------------------------------------------------------------
+
+FROM fleshgrinder/debian:stretch
```

Comment:
This image is already defanged and comes along with a Docker compatible health check script.